sunpy/timeseries/timeseriesbase.py

```diff
@@ -183,8 +191,419 @@ def time_range(self):
         else:
             return None
 
+    @property
+    def channel_info(self):
+        """
+        Information about the channels of the instrument (e.g., wavelengths
+        or energy ranges).
+        """
+        return self._channels
```

Comment:
There is no way to work this out outside of hardcoding it in the subclass?

Reply:
Information about channel wavelengths/energies is typically not in the metadata or the fits headers. The exceptions seem to be Fermi and RHESSI where that information is the literal channel names. For other sources, this information is contained in the individual plot routines (either in axes labels or legend information). I may be missing something, but so far I have not found it in a somewhat common location.